examples/OWNERS.yaml

```diff
@@ -1,5 +1,17 @@
 # For an explanation of the OWNERS.yaml rules and syntax, see:
 # https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
 
-- aghassemi
-- lannka
+- ampproject/wg-access-subscriptions
+- ampproject/wg-ads
+- ampproject/wg-amp4email
+- ampproject/wg-analytics
+- ampproject/wg-approvers
+- ampproject/wg-caching
+- ampproject/wg-codeofconduct
+- ampproject/wg-infra
+- ampproject/wg-outreach
+- ampproject/wg-performance
+- ampproject/wg-runtime
+- ampproject/wg-stories
+- ampproject/wg-ui-and-a11y
+- ampproject/wg-viewers
```

Comment:
I'm not sure we should ping folks on `wg-approvers`, `wg-codeofconduct`, or `wg-outreach` to review changes to tests. Maybe reduce the list to these groups?
```
- ampproject/wg-access-subscriptions
- ampproject/wg-ads
- ampproject/wg-amp4email
- ampproject/wg-analytics
- ampproject/wg-caching
- ampproject/wg-infra
- ampproject/wg-performance
- ampproject/wg-runtime
- ampproject/wg-stories
- ampproject/wg-ui-and-a11y
```

Reply:
I think I am misunderstanding the bot behavior here. Do we ping all these people (if so this is maybe bad)? I thought we only auto ping the `#` prefix?

Reply:
Trimming the list sgtm. We lose a few people if we nix `wg-outreach` that are not covered by the other groups. Maybe @mrjoro can speak for outreach?

Reply:
I believe the bot picks a random owner and adds them as a reviewer. @rcebulko can confirm.

Reply:
ok cool maybe we should do all these as the `?` flavor then.

Reply:
We auto-ping (aka tag/@mention in a comment) anyone with the `#` prefix. All of these people are owners though, and thus all could be assigned to review PRs; ie, they're in the same reviewer pool for changes to files in this directory.

Reply:
If we **do** want all working groups, may as well use `ampproject/working-groups`, which contains the 14 working groups. Note that I don't necessarily fall one way or the other on whether this is the desired ownership

Reply:
The `ampproject/reviewers-amphtml ` team includes anyone who is authorized to merge on this repository.

It seems like this may be a case where it would be useful to start adding some structure to this directory (and `test`)